curiosrc/alertmanager/task_alert.go

```diff
@@ -0,0 +1,395 @@
+package alertmanager
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	logging "github.com/ipfs/go-log/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/lib/harmony/harmonydb"
+	"github.com/filecoin-project/lotus/lib/harmony/harmonytask"
+	"github.com/filecoin-project/lotus/lib/harmony/resources"
+	"github.com/filecoin-project/lotus/node/config"
+	"github.com/filecoin-project/lotus/storage/ctladdr"
+)
+
+var MinimumBalance = types.MustParseFIL("5 FIL")
+
+const AlertMangerInterval = 10 * time.Minute
```

Comment:
This has been dropping. It started out at 1 hour. What's the logic here?
I'm concerned we may slice too thin &  miss out.